### ISSUE TITLE
Syntax highlighting in code blocks

### ISSUE THREAD

**Issue by anukul** [plibither8/markdown-new-tab#36]
Feature request

**Comment by plibither8** [comment]
The markdown parser that I'm using currently doesn't support syntax highlighting. Nevertheless, I understand the need for it. I'll maybe look into other parsers that do offer syntax highlighting. Thanks!

**Comment by anukul** [comment]
Looks like there are some extensions:

- https://github.com/Bloggify/showdown-highlight
- https://github.com/showdownjs/prettify-extension

**Comment by plibither8** [comment]
Yup, I just got to know about the first one! I'll take a look at them :)
